pkg/integrations/manager.go

```diff
@@ -206,7 +206,7 @@ func (m *Manager) runIntegration(ctx context.Context, cfg Config, i Integration)
 		// Apply the config so an instance is launched to scrape our integration.
 		instanceConfig := m.instanceConfigForIntegration(cfg, i)
 		if err := m.im.ApplyConfig(instanceConfig); err != nil {
-			level.Error(m.logger).Log("msg", "failed to apply integration. integration will not run. THIS IS A BUG!", "err", err, "integration", cfg.Name())
+			level.Error(m.logger).Log("msg", "failed to apply integration. integration will not run", "err", err, "integration", cfg.Name())
```

Comment:
This isn't a bug :) If you provide an invalid scrape interval or bad WAL truncation frequency you can run into this.